src/diagrams/mindmap/styles.js

```diff
@@ -1,3 +1,76 @@
-const getStyles = () => ``;
+import { darken, lighten, adjust, invert, isDark } from 'khroma';
 
+const genSections = (options) => {
+  let sections = '';
+
+  for (let i = 0; i < 8; i++) {
+    options['lineColor' + i] = options['lineColor' + i] || options['gitInv' + i];
+    if (isDark(options['lineColor' + i])) {
+      options['lineColor' + i] = lighten(options['lineColor' + i], 20);
+    } else {
+      options['lineColor' + i] = darken(options['lineColor' + i], 20);
+    }
+  }
+
+  for (let i = 0; i < 8; i++) {
+    const sw = '' + (17 - 3 * i);
```

Comment:
```suggestion
    const strokeWidth = 17 - 3 * i;
```

It will be converted to string automatically within the template below.